README.md

```diff
@@ -1,38 +1,77 @@
-<!-- ---
+<!--
+layout: home
 title: Cosmos SDK Tutorials
-description: Tutorials for the Cosmos SDK.
+description: Tutorials for the Cosmos SDK
+sections:
+  - title: Starport Polling App
+    desc: Build a voting application with a web-based UI.
+    url: /starport-polling-app/
+    tag: beginner
+  - title: Starport Blog
+    desc: Learn how Starport works by building a blog.
+    url: /starport-blog/01-index.html
+    tag: beginner
+  - title: Scavenge
+    desc: Users post questions with hashed answers and bounties for whoever can solve them using a commit-reveal scheme that prevents front-running.
+    url: /scavenge/tutorial/01-background.html
+    tag: intermediate
+  - title: Nameservice
+    desc: Build a fully functional naming service on a blockchain.
+    url: /nameservice/tutorial/00-intro.html
+    tag: advanced
```

Comment:
```suggestion
    tag: intermediate
```

Reply:
wdyt? @okwme

Reply:
maybe we use advanced until we have something more advanced and then we reduce it?